tekton/resources/nightly-tests/pipeline-deploy-test-template.yaml

```diff
@@ -61,6 +61,7 @@ spec:
           runAfter: [deploy-pipeline]
           taskRef:
             name: e2e-remote-tests
+          retries: 2
```

Comment:
why the `2` retries?

Reply:
Just in case. I've observed several times flaky tests and want to be on safe side.
Do you prefer 1 retry?